apis/nucleus/src/components/listbox/components/ListBoxSearch.jsx

```diff
@@ -7,46 +7,68 @@ import InstanceContext from '../../../contexts/InstanceContext';
 
 const TREE_PATH = '/qListObjectDef';
 
-export default function ListBoxSearch({ model, keyboard, dense = false, visible = true }) {
+export default function ListBoxSearch({ selections, model, keyboard, dense = false, visible = true }) {
   const { translator } = useContext(InstanceContext);
   const [value, setValue] = useState('');
-  const [isSearching, setIsSearching] = useState(false);
   const theme = useTheme();
 
+  const cancel = () => selections.isActive() && selections.cancel();
+
   const abortSearch = async () => {
-    await model.abortListObjectSearch(TREE_PATH);
-    setIsSearching(false);
+    if (!selections.isModal()) {
+      return;
+    }
+    try {
+      await model.abortListObjectSearch(TREE_PATH);
+    } finally {
+      setValue('');
+    }
   };
 
   useEffect(() => {
-    if (!visible && isSearching) {
-      abortSearch();
+    if (!visible) {
+      cancel(); // cancel and abort when toggling off search
```

Comment:
why cancel selections here?

in client (old filterpane) it is possible to hide the search without cancel selection
why should this work different?